app/components/journalFilterInput/index.tsx

```diff
@@ -17,6 +17,19 @@ interface JournalFilterInputProps {
   onSubmit: (journals: AggregationJournal[]) => void;
 }
 
+const mapJournalSuggestionToAggregationJournal = (j: JournalSuggestion): AggregationJournal => {
+  return {
+    id: j.journalId,
+    title: j.keyword,
+    abbrev: j.abbrev,
+    sci: j.sci,
+    jc: j.jc,
```

Comment:
This word is so abbreviated that it is difficult to understand its meaning.

Please change with more specific words.